osu.Game/Screens/OnlinePlay/OnlinePlayComposite.cs

```diff
@@ -85,9 +92,13 @@ protected override void LoadComplete()
             Playlist.BindCollectionChanged((_, __) => UpdateSelectedItem(), true);
         }
 
-        protected virtual void UpdateSelectedItem()
-            => SelectedItem.Value = RoomID.Value == null || subScreenSelectedItem == null
-                ? Playlist.GetCurrentItem()
-                : subScreenSelectedItem.Value;
+        protected void UpdateSelectedItem()
+        {
+            // null room ID means this is a room in the process of being created.
+            if (RoomID.Value == null)
+                CurrentPlaylistItem.Value = Playlist.GetCurrentItem();
+            else if (subScreenSelectedItem != null)
+                CurrentPlaylistItem.Value = subScreenSelectedItem.Value;
```

Comment:
I personally find this kinda dodgy, but okay.

Reply:
One would hope it wasn't required in the first place, but I'm not at the point I feel confident changing it yet.